drafts/draft-mizrahi-ippm-ioam-flags.xml

```diff
@@ -330,94 +408,125 @@
+
+                      <t>A transit node that processes a packet with the I-bit
+                      set is expected to export the requested IOAM data, and
+                      not incorporate it into the IOAM header.</t>
+
+                      <t>A decapsulating IOAM node that processes a packet
+                      with the I-bit set is expected to export the requested
+                      IOAM data, and decapsulate the IOAM header.</t>
+        </list></t>
 
+        <t>Note that in case of "Immediate Export" being employed,
+        no IOAM trace data is added to the packets traversing the
+        network. As a means to support correlation of exported IOAM
+        data different nodes in the network, a deployment could
+        consider attaching an IOAM E2E option in addition to the
+        trace option, that includes a sequence number. See Bit 1 in
+        the IOAM-E2E-Types. Please refer to 
+		<xref target="I-D.ietf-ippm-ioam-data"/> for a discussion of IOAM data 
+		export and associated formats.</t>
 	</section>
 
-
+	
     <section anchor="IANA" title="IANA Considerations">
-      <t>This document requests the following IANA Actions.</t>
-
-      <section anchor="IANA-Active-Allocation"
-               title="Allocating the Active Flag">
-        <t>IANA is requested to allocate bit 1 in the 
+	
+        <t>IANA is requested to allocate the following bits in the 
 		"IOAM Trace Flags Registry" as follows:</t>
 		
         <t><list style="hanging">
-            <t hangText="Bit 1">"Active" (A-bit)</t>
-          </list></t>
+			<t hangText="Bit 1">"Loopback" (L-bit)</t>
+            <t hangText="Bit 2">"Active" (A-bit)</t>
+			<t hangText="Bit 3">"Immediate Export" (I-bit)</t>
+        </list></t>
 		  
 		<t>Note that bit 0 is the most significant bit in the Flags Registry.
 		</t>
 
-	  </section>
-
     </section>
 
     <section anchor="Performance" title="Performance Considerations">
-      <t>Using active measurement with IOAM requires the use of synthetic 
-	  (overhead) traffic. Obviously such overhead has a cost in terms of the
-	  network bandwidth. Therefore, probe packets should be generated at a 
-	  rate that does not significantly affect the network bandwidth. It should 
-	  be possible for active measurement using replicated packets to be 
-	  performed on a subset of the traffic, so as to prevent significant 
-	  overhead.</t>
+      <t>Each of the three flags that are defined in this document may have
+	  performance implications. When using the loopback mechanism a copy of
+	  the data packet is sent back to the sender, thus generating more traffic
+	  than originally sent by the endpoints. Using active measurement with the 
+	  active flag requires the use of synthetic (overhead) traffic. The 
+	  Immediate Export flag triggers exported packets to be sent to an external 
+	  collector, potentially impacting the collector's performance, as well as 
```

Comment:
I don't think that this is accurate for all the use cases. For example, when the data is exported to a local collector. Would you agree?

Reply:
the comment applies for the next paragraph as well